net.gwyddion.Gwyddion.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "net.gwyddion.Gwyddion",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gwyddion",
+    "copy-icon": true,
+    "rename-icon": "gwyddion",
+    "rename-desktop-file": "gwyddion.desktop",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
does this *actually* work under wayland? It's GTK2 right?

Reply:
Ah, I forgot to test it under wayland. Will do it right away.

Reply:
It works. What is it with GTK2 that you assumed otherwise?

Reply:
Ah now I see it. Even under wayland, if use
` flatpak run --nosocket=x11 net.gwyddion.Gwyddion `
it does not run. But if I use 
`flatpak run --nosocket=wayland net.gwyddion.Gwyddion `
everything works.

Reply:
Should I remove "--socket=wayland"?

Reply:
You should. It'll run under XWayland.

Reply:
Done